discovery-provider/alembic/versions/92571f94989a_add_trending_views.py

```diff
@@ -0,0 +1,290 @@
+"""add trending views
+
+Revision ID: 92571f94989a
+Revises: cbf0fafabdeb
+Create Date: 2021-09-22 16:27:23.281441
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '92571f94989a'
+down_revision = 'cbf0fafabdeb'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    connection = op.get_bind()
+    connection.execute(
+        """
+        DROP MATERIALIZED VIEW IF EXISTS aggregate_interval_plays;
```

Comment:
How long does this migration take to run on a db? Wanna make sure it doesnt cause an infinite restart loop if kube probe timeouts that are too low

Reply:
Yeah, i'd also be curious to see some timing stats from one of the sandbox discovery nodes. You maybe did that already, but this seems like it's going to be extremely expensive

Reply:
Using a snapshot prod DB on a db.m5.xlarge, it takes ~54 seconds to create the aggregate interval plays mat view and ~11 sec for the trending_params mat view - on a DB w/ no load

Reply:
it's in k8s manifests -- 
https://github.com/AudiusProject/audius-k8s-manifests/blob/57e933210b8ae2e08e1e17192d1c03dc0b889eab/audius/discovery-provider/discovery-provider-deploy.yaml#L41

300s initially, then 50s to fail

Reply:
How fast would it be if we make the matview here with no data and refresh it later in the indexing flow (like we're doing already)?

> You can also instruct PostgreSQL not load any data into the materialized view while you are creating it by specifying the WITH NO DATA option